src/filerepository.js

```diff
@@ -205,6 +226,25 @@ export default class FileRepository extends Plugin {
 
 		this.loaders.remove( loader );
 	}
+
+	/**
+	 * Registers or deregisters pending action bound with upload progress.
+	 *
+	 * @private
+	 */
+	_updatePendingAction() {
+		const pendingActions = this.editor.plugins.get( PendingActions );
+		const getMessage = value => `Upload in progress ${ parseInt( value ) }%.`;
```

Comment:
Translate service should be used.

Reply:
Also, you don't need to define it every time. Only if you plan to add peding action, so I would move it inside `if` below, closer to the place where it is used.

Reply:
This is a function.

Reply:
Or you mean that this function doesn't have to be defined every time?

Reply:
But it shoudl be moved to inside if below. 👍